cmd/stacker/main.go

```diff
@@ -29,6 +29,7 @@ var (
 )
 
 //go:embed lxc-wrapper/lxc-wrapper
+//go:embed stacker-bom
```

Comment:
Is there a reason that you would stuff a go binary into another go binary?
Why not just import github.com/project-stacker/stacker-bom/pkg and use that ?

Reply:
@rchincha was there a reason to go with go:embed here?

(Did you want to process it with m4?  I kid!)